node/pegnet/transaction.go

```diff
@@ -0,0 +1,60 @@
+package pegnet
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+const (
+	DefaultPad = 3
+)
+
+// SplitTxID splits a TxID into it's parts.
+// TxID format : [TxIndex]-[EntryHash]
+//				 1-c99dedea0e4e0c40118fe7e4d515b23cc0489269c8cef187b4f15a4ccbd880be
+func SplitTxID(txid string) (index int, entryhash string, err error) {
+	arr := strings.Split(txid, "-")
+	if len(arr) != 2 {
```

Comment:
hashes that don't have a `#-` should still be valid

Reply:
Should an entryhash be considered a "txid"?

I was under the impression that you either use the entryhash or txid. An entryhash is not a txid.

Reply:
entryhash is already a catchall that supports factom txids (for burns)

Reply:
Oh man. I forgot about burns, so `txid` could be a pegnet txid, or a factoid one. But the factoid burn would be queried `0-<factoid txid>`.

I don't think we should use the same word for a hash without the prefix.